apps/nuxt3-ssr/components/content/ContentBlockNetwork.vue

```diff
@@ -14,8 +14,8 @@ defineProps<{
         :imageUrl="network?.logo?.url"
         :title="network?.name"
         :description="network?.description"
-        url="#"
-        :links="[{ title: 'Read more', url: '#' }]"
+        :url="network.website"
+        :links="[{ title: 'Read more', url: network.website }]"
```

Comment:
why are there two parameters, i.e., is :links not all you need?

Reply:
Ah, I see, the second link might go to an internal page, but that doesn't exist yet.

Reply:
yea that it , decided to use the url for now